packages/app-bridge/src/AppBridgePlatformApp.ts

```diff
@@ -50,24 +50,22 @@ type AppBaseProps = {
     token: string;
     marketplaceServiceAppId: string;
     connected: boolean;
+    settings: { [key: string]: string };
```

Comment:
Should not this be an array?

Reply:
if im not wrong the buildableSettings stores it like {"key-one": "value-one", "key-two": "value-two"} every entry in a object

Reply:
Yes, you're right.